### PR TITLE
Make light theme illegal

### PR DESCRIPTION
**Pull request by Bowser65**
btw have i told you  u cute?

### CODE REVIEW

**Comment by james121op** [comment]
merge it already don't delay

**Comment by Devoxin** [comment]
do it u wont

File: src/views/terms.ejs [hunk 1]
```diff
@@ -21,6 +21,10 @@
   <p>You agree not to disclose your password to any third party. You must notify us immediately upon becoming aware of any breach of security or unauthorized use of your account.</p>
 
 
+  <h2>Rights to use the Service</h2>
+  <p>You accept to not (and not attempt to) (i) use the "Light theme" provided by Discord, (ii) prevent in any way the posting of analytics to our endpoints, (iii) say thay you are not cute.</p>
```
Comment (Devoxin) [inline src/views/terms.ejs]: > (iii) say thay you are not cute.</p>
`that`

Comment (Bowser65) [reply src/views/terms.ejs]: make a suggestion i'm too lazy to correct it

Comment (Devoxin) [reply src/views/terms.ejs]: no, put in the work, earn ur title of member cnt

**Comment by Devoxin** [comment]
looks good, LGTM

**Comment by TurtleGamingFTW** [comment]
I don't know about you guys but light theme is pretty good

**Comment by ohlookitsderpy** [comment]
[**@TurtleGamingFTW**](https://github.com/TurtleGamingFTW) commented on [Jul 15, 2019, 5:48 PM GMT+1](https://github.com/powercord-org/powercord-backend/pull/18#issuecomment-511480360 "2019-07-15T16:48:42Z - Replied by Github Reply Comments"):
> I don't know about you guys but light theme is pretty good

im calling the fbi

**Comment by ghostlydot** [comment]
> 
> 
> [**@TurtleGamingFTW**](https://github.com/TurtleGamingFTW) commented on [Jul 15, 2019, 5:48 PM GMT+1](https://github.com/powercord-org/powercord-backend/pull/18#issuecomment-511480360):
> 
> > I don't know about you guys but light theme is pretty good
> 
> im calling the fbi

And I'mma calling the FBI on you

**Comment by TurtleGamingFTW** [comment]
> [**@TurtleGamingFTW**](https://github.com/TurtleGamingFTW) commented on [Jul 15, 2019, 5:48 PM GMT+1](https://github.com/powercord-org/powercord-backend/pull/18#issuecomment-511480360):
> 
> > I don't know about you guys but light theme is pretty good
> 
> im calling the fbi

Some people like having their eyes burned

**Comment by Bowser65** [comment]
light theme must die

**Comment by TurtleGamingFTW** [comment]
nou

**Comment by ghostlydot** [comment]
No Light Theme must live

**Comment by Bowser65** [comment]
alternative answer

**Comment by ghostlydot** [comment]
😭 I thought we were friends?

**Comment by TurtleGamingFTW** [comment]
You'll be hearing from my lawyers if you make light theme illegal

**Comment by MrLar** [comment]
Why does this thread exist

**Comment by Bowser65** [comment]
Because dumb people invented light theme

**Comment by MrLar** [comment]
But some people need it because of their medical condition therefore making it illegal would be illegal by itself mmLol

**Comment by ghostlydot** [comment]
I think bowser just needs a hug

**Comment by Bowser65** [comment]
That's a smart answer lars. We don't do smart here

**Comment by MrLar** [comment]
He has a girlfriend dw about it he gets more than hug, or maybe he doesnt get enough of that 🤔

**Comment by ghostlydot** [comment]
This is the best pull request discussion i have ever been a part of.

It is also the only pull request discussion i have ever been a part of.

**Comment by TurtleGamingFTW** [comment]
It's also my first time mmLol

**Comment by GreemDev** [comment]
this thread is a large mmLol

**Comment by LandenStephenss** [comment]
this def needs to happen :eyes:

**Comment by TurtleGamingFTW** [comment]
Light is good though

**Comment by KorbsStudio** [comment]
You girls make up already and merge this bad boy

**Comment by abyssal512** [comment]
I was expecting it so Powercord removed the light theme from application options, yet another let down 😔

**Pull request closed without merging**